Sources/UIKitExtensions/UICollectionReusableView+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UICollectionReusableView.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+import UIKit
+public extension ConfigurationSet where Base: UICollectionReusableView {
+    
+    func apply(_ layoutAttributes: UICollectionViewLayoutAttributes) -> ConfigurationSet<Base> {
+        return set { ( UICollectionReusableView: Base ) in
+            UICollectionReusableView.apply(layoutAttributes)
+        }
+    }
+    
+    @available(iOS 8.0, *)
+    func preferredLayoutAttributesFitting(_ layoutAttributes: UICollectionViewLayoutAttributes) -> ConfigurationSet<Base> {
```

Comment:
This function does not set anything during View Setup, it is used during lay outing collection view therefore we don't need it.